src/main/java/com/tikal/jenkins/plugins/multijob/views/MultiJobView.java

```diff
@@ -152,6 +152,8 @@ private void addProjectFromBuilder(MultiJobProject project,
             Item tli = Jenkins.getInstance().getItem(projectConfig.getJobName(), project.getParent(), AbstractProject.class);
             if (tli instanceof MultiJobProject) {
                 MultiJobProject subProject = (MultiJobProject) tli;
+                // TODO: only match those that have the same Alias name
```

Comment:
It is not clear what this todo means. There is alias comparison below. Would you please rephrase?